examples/seismic/benchmark.py

```diff
@@ -273,15 +269,22 @@ def plot(problem, **kwargs):
     with RooflinePlotter(title=title, figname=figname, plotdir=resultsdir,
                          max_bw=max_bw, flop_ceils=flop_ceils,
                          fancycolor=True, legend=legend) as plot:
-        for key, gflopss in gflopss.items():
-            oi_value = oi[key]
-            time_value = time[key]
-            key = dict(key)
-            run = (key["dse"], key["dle"])
-            label = "<%s,%s>" % run
-            oi_loc = 0.05 if len(str(key["space_order"])) == 1 else 0.06
-            oi_annotate = {'s': 'SO=%s' % key["space_order"],
-                           'size': 4, 'xy': (oi_value, oi_loc)} if run[0] else None
+        for k, v in gflopss.items():
+            so = dict(k)['space_order']
+
+            oi_value = oi[k]
+            time_value = time[k]
```

Comment:
Couldn't these two just be inlined in the function call later? It would be more concise that way.

Reply:
👍

Reply:
Oh, now I remember. The reason I prefer to make it explicit is that the suffix ``_value`` helps me remembering that those are the (x,y) **values** in the plot. ``oi[k]`` and ``time[k]`` aren't very informative